test/lib/s3_test.exs

```diff
@@ -29,7 +29,7 @@ defmodule ExAws.S3Test do
         "bucket",
         headers: %{"x-amz-request-payer" => "requester"},
         prefix: "/path/to/objs",
-        start_after: "/path/to/objs/sub",
+        start_after: "/path/to/objs/sub"
```

Comment:
This PR includes `mix format` diff a06c3c133edfaf6f37a3cb503bee99c573d58a92. I'll comment about where I changed intended.